### ISSUE TITLE
panic on boot after merging kdb patchset

### ISSUE THREAD

**Issue by emaste** [FreeBSDFoundation/freebsd#2]
```
panic: Invalid instruction length in exception
KDB: stack backtrace:
db_trace_self() at db_trace_self_wrapper+0x30
         pc = 0xffffff80003c3370  lr = 0xffffff8000014c08
         sp = 0xffffff80685b26e0  fp = 0xffffff80685b2810

db_trace_self_wrapper() at vpanic+0xc8
         pc = 0xffffff8000014c08  lr = 0xffffff8000153e30
         sp = 0xffffff80685b2820  fp = 0xffffff80685b2890

vpanic() at kassert_panic+0x164
         pc = 0xffffff8000153e30  lr = 0xffffff8000153d64
         sp = 0xffffff80685b28a0  fp = 0xffffff80685b2970

kassert_panic() at do_el1h_sync+0x30
         pc = 0xffffff8000153d64  lr = 0xffffff80003cd91c
         sp = 0xffffff80685b2980  fp = 0xffffff80685b29a0

do_el1h_sync() at handle_el1h_sync+0x54
         pc = 0xffffff80003cd91c  lr = 0xffffff80003c3854
         sp = 0xffffff80685b29b0  fp = 0xffffff80685b2b70

handle_el1h_sync() at fork_exit+0xb0
         pc = 0xffffff80003c3854  lr = 0xffffff800011efdc
         sp = 0xffffff80685b2b80  fp = 0xffffff80685b2bb0

fork_exit() at fork_trampoline+0x10
         pc = 0xffffff800011efdc  lr = 0xffffff80003cd788
         sp = 0xffffff80685b2bc0  fp = 0x00000000

KDB: enter: panic
```

**Comment by zxombie** [comment]
Is this fixed by 3b733d4bad1cff7167228f908be21df0219252d2?

**Comment by emaste** [comment]
Fixed by 3b733d4
